apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+    <div class="multiple-provider-form">
+        <p-field-title class="title">
+            Provider
+        </p-field-title>
+        <div class="radio-container">
+            <p-radio-group>
+                <p-radio v-for="provider in state.providerList"
+                         :key="provider.name"
+                         :selected="state.selectedProvider"
+                         :value="provider.name"
+                         class="provider-item"
+                         @change="handleChangeProvider"
+                >
+                    <div class="content-menu-item">
+                        {{ provider.label }}
+                    </div>
+                </p-radio>
+            </p-radio-group>
+        </div>
+        <div class="dropdown-container">
+            <p-select-dropdown :selected="state.selectedProvider"
+                               :items="state.providerList"
+                               class="select-dropdown"
+                               @update:selected="handleChangeProvider"
+            >
+                <template #menu-item--format="{ item }">
+                    <div class="content-menu-item">
+                        <span>{{ item.label }}</span>
+                    </div>
+                </template>
+            </p-select-dropdown>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { computed, reactive } from 'vue';
+
+import {
+    PFieldTitle, PRadioGroup, PRadio, PSelectDropdown,
+} from '@spaceone/design-system';
+
+
+import { store } from '@/store';
+
+import type { ProviderReferenceMap } from '@/store/modules/reference/provider/type';
+
+import {
+    useCollectorFormStore,
+} from '@/services/asset-inventory/collector/shared/collector-forms/collector-form-store';
+
+const collectorFormStore = useCollectorFormStore();
+const collectorFormState = collectorFormStore.$state;
+
+
+const state = reactive({
+    providers: computed<ProviderReferenceMap>(() => store.getters['reference/providerItems']),
+    providerList: computed(() => {
+        const supportedProviderList = collectorFormState.repositoryPlugin?.capability?.supported_providers ?? [];
+        return supportedProviderList.map((providerName) => ({
+            label: state.providers[providerName]?.name,
+            name: providerName,
+        }));
+    }),
+    selectedProvider: computed(() => collectorFormState.provider),
+});
+
+const handleChangeProvider = (provider) => {
+    collectorFormStore.setProvider(provider);
+};
+
+
+(() => {
+    collectorFormStore.setProvider(state.providerList[0].name);
```

Comment:
nitpick: To handle the case where the `provider` cannot be retrieved for any reason...

```typescript
collectorFormStore.setProvider(state.providerList?.[0].name);
```